src/containers/Permission.jsx

```diff
@@ -36,18 +53,41 @@ const Permission = ({ match }) => {
             Application: {appName.toUpperCase()}
           </Typography>
           <Typography variant="h2">Permission: {permissionName}</Typography>
-          <List>
-            {queryResult.data[0].attributes.permissions[permissionName]
-              .verbs ? (
-              queryResult.data[0].attributes.permissions[
-                permissionName
-              ].verbs.map(permission => (
-                <ListItemText key={permission.name}>{permission}</ListItemText>
-              ))
-            ) : (
-              <ListItemText>ALL</ListItemText>
-            )}
-          </List>
+          <Typography variant="h3">
+            {queryResultApps.data.length > 0
+              ? !queryResultApps.data[0].attributes.permissions[permissionName]
+                  .verbs ||
+                (queryResultApps.data[0].attributes.permissions[permissionName]
+                  .verbs.length > 1 &&
+                  queryResultApps.data[0].attributes.permissions[
+                    permissionName
+                  ].verbs.indexOf('GET') >= 0)
+                ? 'Lecture et Écriture'
+                : queryResultApps.data[0].attributes.permissions[permissionName]
+                    .verbs.length === 1 &&
+                  queryResultApps.data[0].attributes.permissions[permissionName]
+                    .verbs[0] === 'GET'
+                ? 'Lecture'
+                : 'Écriture'
+              : !queryResultKonnectors.data[0].attributes.permissions[
+                  permissionName
+                ].verbs ||
+                (queryResultKonnectors.data[0].attributes.permissions[
+                  permissionName
+                ].verbs.length > 1 &&
+                  queryResultKonnectors.data[0].attributes.permissions[
+                    permissionName
+                  ].verbs.indexOf('GET') >= 0)
+              ? 'Lecture et Écriture'
+              : queryResultKonnectors.data[0].attributes.permissions[
+                  permissionName
+                ].verbs.length === 1 &&
+                queryResultKonnectors.data[0].attributes.permissions[
+                  permissionName
+                ].verbs[0] === 'GET'
+              ? 'Lecture'
```

Comment:
for code readability, could we extract a method called "displayRight" or "displayPermission" that take a value, check if it equals to "GET" and return 'lecture' or 'ecriture' ?